src/export_from_inboveg/HT3260_data_preparation.Rmd

```diff
@@ -0,0 +1,297 @@
+  select(survey, recording_givid, user_reference, var, value_class = var_description_or_value)
+  
+structure_num_elu <- structure_3260 %>% 
+  filter(var_code %in% var_num_elu) %>% 
+  mutate(value_num = as.numeric(str_replace(value, ",", "\\.")),
+         var = ifelse(is.na(var_description_or_value), var_value_code, var_description_or_value)) %>% 
+  filter(!is.na(value_num)) %>% 
+  select(survey, recording_givid, user_reference, var, value_num)
+
+# tmp - to do: opvolgen of NA's in description aangevuld zijn
+structure_3260 %>% 
+  filter(is.na(var_description_or_value),
+         !(var_value_code %in% c("SDiep", "SStr"))) %>% 
+  select(survey, recording_givid, user_reference, var_value_code, var_description_or_value) # 1 record
+
+structure_num <- structure_3260 %>% 
+  filter(var_code %in% var_num) %>% 
+  mutate(value_num = as.numeric(var_description_or_value)) %>% 
+  select(survey, recording_givid, user_reference, var, value_num)
+  
+structure_multiple <- structure_3260 %>% 
+  filter(var_code %in% var_multiple,
+         !(var_description_or_value %in% c("Secchi-diepte in m", "Secchi-diepte meer dan (m)"))) %>%
+  select(survey, recording_givid, user_reference, var, value_class = var_description_or_value)
+# to do: wijze hoort mss bij header; maar is multiple value (max 3)
+
+structure <- bind_rows(structure_class, structure_multiple, structure_num_elu, structure_num)
+
+```
+
+layerqual
+```{r}
+layerqual_3260 %>% 
+  select(value, value_perc) %>% 
+  arrange(value) %>% distinct()
+# to do: 0,1 in inboveg laten aanpassen, want levert NA op
+
+layerqual <- layerqual_3260 %>% 
+  mutate(value_num = as.numeric(value_perc)) %>% 
+  select(survey, recording_givid, user_reference, var, value_num)
+# values < 1% are converted to 0.5 %
+
+```
+
+combined: site = structure + layer
+```{r}
+site <- bind_rows(structure, layerqual)
+  
+```
+
+@Toon: opbouw data.frame 'site' ok? of value_num & value_class te combineren?
+
+## vegetation
+rearrangement of vegetation
+```{r}
+vegetation_3260 %>% 
+  filter(is.na(name_scientific)) %>% 
+  count(name_original) %>% arrange(name_original)
+
+sel_species <- c("Enteromorpha sp.")
+sel_species <- c("Salix 'alba' groep")
+sel_species <- c("Poaceae", "poaceae")
+
+vegetation_3260 %>% 
+  filter(name_original %in% sel_species) %>% 
+  left_join(select(header, recording_givid, user_reference), by = "recording_givid") %>% 
+  select(recording_givid, user_reference, name_original, species_cover_code) 
+  
+# soorten te groeperen: to do (later)
+#   Callitriche 'obtusangula/platycarpa' groep en Callitriche platycarpa/obtusangula   -
+#   darmwier   en Enteromorpha sp.
+# gras (poaceae, vegetatief, niet gedetermineerd),   gras vegetatief en Poaceae, poaceae
+# mos en mos niet gedetermineerd
+# Salix 'alba' groep en Salix smalbladig en wilg smal
+
+vegetation <- vegetation_3260 %>% 
+  mutate(name = ifelse(is.na(name_scientific), name_original, name_scientific)) %>% 
```

Comment:
Nederlands naam zou ook handig zijn

Reply:
via get_inboveg_recording() kan je de Nederlandse naam niet opvragen; bij mijn weten zit dit niet in inboveg; mss na te vragen aan Sophie of Pieter